### PR TITLE
Conversion fix

### PR DESCRIPTION
**Pull request by jSydorowicz21**
Fixed a conversion bug with the last optimizations push

### CODE REVIEW

File: ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java [hunk 1]
```diff
@@ -74,13 +72,13 @@ public String deleteDrink(String id) {
     public List<LambdaDrink> getAllDrinks() {
         EndpointUtility endpointUtility = new EndpointUtility();
         String response = endpointUtility.getEndpoint(DRINKS_ENDPOINT);
-        List<LambdaDrink> lambdaDrinks;
+        List<String> lambdaDrinks;
```
Comment (jnfamousdan) [inline ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java]: should this be final?

Comment (jSydorowicz21) [reply ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java]: Yes, converted class to finals

File: ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java [hunk 2]
```diff
@@ -61,7 +60,7 @@ public LambdaDrink updateDrink(LambdaDrink lambdaDrink) {
     }
 
     public String deleteDrink(String id) {
-        EndpointUtility endpointUtility = new EndpointUtility();
+        final EndpointUtility endpointUtility = new EndpointUtility();
```
Comment (jnfamousdan) [inline ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java]: these are repeating code lines - perhaps make this a class variable?

File: ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java [hunk 3]
```diff
@@ -70,9 +69,9 @@ public String deleteDrink(String id) {
     }
 
     public List<LambdaDrink> getAllDrinks() {
-        EndpointUtility endpointUtility = new EndpointUtility();
-        String response = endpointUtility.getEndpoint(DRINKS_ENDPOINT);
-        List<String> lambdaDrinks;
+        final EndpointUtility endpointUtility = new EndpointUtility();
```
Comment (jnfamousdan) [inline ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java]: same as previous repeating code

**Review by jnfamousdan — APPROVED** [review]
approving with repeating codelines!

File: ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java (hunk 2 above)
Comment (jSydorowicz21) [reply ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java]: Made global variable

File: ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java (hunk 3 above)
Comment (jSydorowicz21) [reply ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java]: See above

**Pull request merged**